tests/client/test_client_task_meilisearch.py

```diff
@@ -66,3 +67,23 @@ def test_get_task_inexistent(client):
     """Tests getting a task that does not exists."""
     with pytest.raises(Exception):
         client.get_task("abc")
+
+
+def test_cancel_tasks(client):
+    """Tests cancel a task with uid 1."""
+    task = client.cancel_tasks({"uids": 1})
+    assert isinstance(task, TaskInfo)
+    assert task.task_uid is not None
+    assert task.index_uid is None
+    assert task.status == "enqueued"
```

Comment:
You should not test this as it could be processing/processed by the time you do the assertion